app/resources/views/pages/contact/form.blade.php

```diff
@@ -24,6 +24,8 @@
 				<div class="col-xs-12">
 					<input type="submit" class="clean" value="Send Message">
 				</div>
+				<div class="col-xs-12">
+					<a href="https://www.surveymonkey.com/r/2SZ98H7">Please give us a survey. Thank You!</a>
```

Comment:
I like how simple this is.

Can you add more space to separate the link from the "Send Message" button?  This would involve updating resources/assets/sass/_contact.scss.  After saving changes there, run "gulp" before reloading the contact page to see the results. gulp will convert that .scss file into CSS.

The link looks so close to the "Send Message" button that it looks like a description of what clicking the "Send Message" button will do.  The extra space could help make it look like a separate clickable link.
![image](https://user-images.githubusercontent.com/11838191/42446477-6a15ad3a-8344-11e8-939c-f514295aedd9.png)